### PR TITLE
[SMALL] Mention migration operation in SQLite NotSupportedException

### LINKED ISSUE #5389: Migrations: Include the operation type in SQLite "SQLite cannot support this migration operation" message
And point to the limitations doc.

### PR DESCRIPTION
**Pull request by rowanmiller**
Fixes #5389

### CODE REVIEW

**Comment by rowanmiller** [comment]
@natemcmaster could you take a quick look at this one

File: test/Microsoft.EntityFrameworkCore.Sqlite.Tests/Migrations/SqliteMigrationSqlGeneratorTest.cs [hunk 1]
```diff
@@ -262,55 +262,55 @@ public void RestartSequenceOperation_not_supported()
         public override void AddForeignKeyOperation_with_name()
         {
             var ex = Assert.Throws<NotSupportedException>(() => base.AddForeignKeyOperation_with_name());
-            Assert.Equal(SqliteStrings.InvalidMigrationOperation, ex.Message);
+            Assert.Equal(SqliteStrings.InvalidMigrationOperation(nameof(AddForeignKeyOperation)), ex.Message);
```
Comment (natemcmaster) [inline test/Microsoft.EntityFrameworkCore.Sqlite.Tests/Migrations/SqliteMigrationSqlGeneratorTest.cs]: Yay for `nameof`!

**Comment by natemcmaster** [comment]
:shipit:

**Comment by rowanmiller** [comment]
Merged https://github.com/aspnet/EntityFramework/commit/4b23c5db78221691669d33a4577690f0ae37ea9b

**Pull request closed without merging**